Zero-Gravity-Store-React/src/App.js

```diff
@@ -1,42 +1,29 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Footer } from "./Components/Footer/Footer";
```

Comment:
You can add all imports in a single file in components and then import that single file where you need it.